pkg/runtime/kubernetes/utils.go

```diff
@@ -28,36 +28,42 @@ import (
 	"github.com/sealerio/sealer/utils/ssh"
 
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 )
 
-// VersionCompare :if v1 >= v2 return true, else return false
-func VersionCompare(v1, v2 string) bool {
+//KubeVersion is a Metadata kubernetes version field such as "v1.19.8"
+type KubeVersion struct {
+	FieldList []string
+}
+
+// Version swap string to []string
```

Comment:
Version takes version string, and encapsulates it in KubeVersion.
The param filed "v1 string" => "version string" could be better.